skycoin/skywire
# Implement a UDP transport

Did you run `make format && make check`? Yes

Fixes #391

 Changes:	
- Implement a UDP transport

How to test this PR:
- Checkout `skywire-services` against `feature/sudp`
- Run integration build
- Run the generic integration environment
- Run `make integration-startup-sudp`
- Run `curl --data {'"recipient":"'$PK_A'", "message":"Hello"}' -X POST $CHAT_C`
- Make sure that message appeared in `VisorA` logs
